group-coordinator/src/test/java/org/apache/kafka/coordinator/group/streams/StreamsCoordinatorRecordHelpersTest.java

```diff
@@ -300,12 +300,13 @@ public void testNewStreamsGroupEpochRecord() {
                 .setGroupId(GROUP_ID),
             new ApiMessageAndVersion(
                 new StreamsGroupMetadataValue()
-                    .setEpoch(42),
+                    .setEpoch(42)
+                    .setMetadataHash(42),
```

Comment:
The only line where you put 42?

Reply:
Yeah, I followed epoch value to check we can get expected metadata hash with `StreamsCoordinatorRecordHelpers.newStreamsGroupEpochRecord`. It's like `GroupCoordinatorRecordHelpersTest#testNewShareGroupEpochRecord`.